src/app-layout/__tests__/trigger-button.test.tsx

```diff
@@ -519,8 +519,7 @@ describe('Visual Refresh Toolbar trigger-button', () => {
         ).toBe(false);
         expect(wrapper!.findByClassName(toolbarTriggerButtonStyles['trigger-tooltip'])).toBeNull();
         expect(() => getByText(mockTooltipText)).toThrow();
-        fireEvent.focus(wrapper!.getElement());
-
+        fireEvent.focus(wrapper!.getElement(), mockEventBubbleWithRelatedTarget);
```

Comment:
I didn't get what's the purpose of the related target mock. Could you elaborate on that?

Reply:
There will be no relatedTarget on the event when split panel is closed via the close button on the split panel, but there is one for when a user opens or closes via the other buttons or keyed navigation.